lib/transloadit.rb

```diff
@@ -76,6 +77,25 @@ def assembly(options = {})
   end
 
   #
+  # Gets user billing reports for specified month and year.
+  # Defaults to current month or year if corresponding param is not specified.
+  #
+  # @param [Integer] month the month for which billing reports should be retrieved.
+  #   defaults to current month if not specified.
+  # @param [Integer] year the year for which billing reports should be retrieved.
+  #   defaults to current year if not specified.
+  #
+  def bill(month = nil, year = nil)
+    today = Date.today
+    # convert month to 2 digit  format
+    month = format('%02d', month || today.month)
+    year  = today.year if year.nil?
```

Comment:
:art: indentation is a little off here, year only needs once space before the equal sign. Yep, we should automate that style check somewhen :)